src/install_rules.ml

```diff
@@ -244,6 +244,9 @@ module Gen(P : Install_params) = struct
         (Utils.install_file ~package ~findlib_toolchain:ctx.findlib_toolchain)
     in
     let entries = local_install_rules entries ~package in
+    SC.add_alias_deps sctx
+      (Alias.package_install ~context:ctx ~pkg:package)
+      (List.map entries ~f:(fun (e : Install.Entry.t) -> e.src));
```

Comment:
Shouldn't we be using the destination of the symlink however? Otherwise, the symlink will not be made.

Reply:
That's what we are doing. `local_install_rules` replaces all the `src` fields by the symlinks